lib/response_bank.rb

```diff
@@ -67,6 +77,23 @@ def cache_key_for(data)
       end
     end
 
+    def check_encoding(env, default_encoding = 'br')
+      # Cloudflare now supports brotli, gzip or uncompressed
+      if env.key?('HTTP_ACCEPT_ENCODING')
+        if env['HTTP_ACCEPT_ENCODING'] == 'gzip'
+          env['response_bank.encoding'] = 'gzip'
+          env['response_bank.server_cache_encoding'] = 'gzip'
```

Comment:
The two env settings of `response_bank.encoding` and `response_bank.server_cache_encoding` are confusing as I read through the code. It's clear we want to know what algo to compress the object at rest, and if the client supports this algo. However, I'm inclined to return to the previous logic and use a boolean state in the middleware (line 16) to determine if the content should be uncompressed. functionally `uncompressed_output =  env['HTTP_ACCEPT_ENCODING'].to_s.include?(env('response_bank.server_cache_encoding')` or something like this.